utils/discord-sender.js

```diff
@@ -1,16 +1,35 @@
+// @flow
 import request from 'request-promise';
 import config from '../config.json';
 import logger from '../utils/logger';
+import {translate} from '../i18n';
 
-function formatDiscordDescription(event) {
-    return `**Quando:** ${event.eventDate}
-**Dove:** ${event.location}
-**Chiusura application:** ${event.applicationClose}`;
+function formatDiscordDescription(event: $FlowFixMe, language: string = 'en') {
+    const t = translate(language);
+    return `**${t('when', 'When:')}** ${event.eventDate}
+**${t('where', 'Where:')}** ${event.location}
+**${t('closing', 'Applications close:')}** ${event.applicationClose}`;
 }
 
-export default function postEvent(event) {
+export default function postEvent(event: $FlowFixMe) {
     logger.info('Posting event to discord: ', event);
+    const t = translate(config.language);
+    const description = formatDiscordDescription(event, config.language);
+    const content = `${t(
```

Comment:
I think this is not so much readable.
I thought two ways:
* Template tag ``i18n`Key ${1} key ${2}` -> i18n([ 'Key ', ' key ', '' ], 1, 2 )``
* printf-like syntax

What do you think about?

Reply:
I understand it's not perfectly readable. Instead of a template syntax, I would use interpolation (which needs to be developed for this translation system), which would be used like this in this scenario:
```javascript
const content = t(
  'content',
  { /* object with variables */ },
  'default message'
);
```

Template strings are cool too, but that would require using weird strings as a key for language dictionaries, for example the string `'Applications will close in: *'`.
In addition to this, using a template string still forces you to keep the position of keys and variables inside the string for every language, which I am not sure is the case for all languages for this sentence.

Reply:
@Takeno I added a commit right now with the new interpolation function. Does it look better now? I feel like this is the best solution because it is now possible to have the placeholders in different spots for each language, in case it's needed.

Meanwhile, I also updated the README with informations on how to configure languages.